gnovm/cmd/gno/util.go

```diff
@@ -105,6 +106,89 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+// targetFromPatterns returns a list of target paths that match the patterns.
+// Each pattern can represent a file or a directory, and if the pattern
+// includes `/...“, it signifies a directory search.
```

Comment:
```suggestion
// includes "/...", the "..." is treated as a wildcard, matching any string,
// including strings with slashes and the empty string.
```

(reusing the description by `go help packages`) :)

Reply:
Resolved-by: d16ed9ec1b491cb90bdf3b7356497668bf01e4da
Thanks!